host/cli/download.go

```diff
@@ -24,16 +24,17 @@ func init() {
 	}
 
 	Register("download", runDownload, `
-usage: flynn-host download [--driver=<name>] [--root=<path>] [--repository=<uri>] [--tuf-db=<path>] [--manifest-dir=<dir>]
+usage: flynn-host download [--driver=<name>] [--root=<path>] [--repository=<uri>] [--tuf-db=<path>] [--config-dir=<dir>] [--bin-dir=<dir>]
 
 Options:
   -d --driver=<name>       image storage driver [default: aufs]
   -r --root=<path>         image storage root [default: /var/lib/docker]
   -u --repository=<uri>    image repository URI [default: https://dl.flynn.io/images]
```

Comment:
Maybe this should be `https://dl.flynn.io/tuf`? `images` collides with other things like VM images